frontend/src/metabase/admin/settings/components/SettingHeader/SettingHeader.tsx

```diff
@@ -0,0 +1,39 @@
+import type React from "react";
+
+import { Box, Text } from "metabase/ui";
+
+export const SettingHeader = ({
```

Comment:
!nit, would be nice to spread the rest of the props on the parent nodes (e.g. `<Box {...props}>`) in both of these components so folks can do things like attach `data-testid` etc.